cg/cli/upload/delivery_report.py

```diff
@@ -138,28 +146,41 @@ def _update_delivery_report_date(status_api, case_id):
         _suggest_cases_delivery_report(context)
         context.abort()
 
+    status_api = context.obj["status"]
+
+    if not analysis_started_at:
+        analysis_started_at = status_api.family(family_id).analyses[0].started_at
+        LOG.debug("using analysis date: %s", analysis_started_at)
```

Comment:
According to coveralls, the if-statement is covered, but not the following two lines. 
Could we benefit from a test-case for when analysis date is not explicitly specified?

Reply:
should be fixed here: test_analysis_without_started_at